projects/plugins/boost/app/admin/class-admin.php

```diff
@@ -30,6 +30,7 @@ class Admin {
 	 * Nonce action for setting the status of show_rating_prompt.
 	 */
 	const SET_SHOW_RATING_PROMPT_NONCE = 'set_show_rating_prompt';
+	const SET_SHOW_SCORE_PROMPT_NONCE  = 'set_show_score_prompt';
```

Comment:
I was thinking about this on my walk earlier too - we can probably refine it so that we have

- ShowModal
- Message

As it stands, both the rating prompt and the score prompt run from a change in scores (which are checked twice) so code could certainly be made more DRY